.gitattributes

```diff
@@ -105,11 +105,8 @@
 *.pvr               binary
 *.snk               binary
 *.tga               binary
-*.tif               binary
```

Comment:
This change comes from the automatic copy from the shared-infrastructure submodule when building. 

I think you need to rebuild the solution and push again making sure all submodules are up-to-date and that the root `.gitattributes` and `.editorconfig` files are updated from the submodule on build.

Reply:
I can do this if you want?

Reply:
Reverted this change.